code/modules/chemistry/tools/dispensers.dm

```diff
@@ -80,6 +80,20 @@
 			return TRUE
 		return ..()
 
+	proc/bolt_unbolt(mob/user)
+		if(!src.anchored)
+			var/turf/T = get_turf(src)
+			if (istype(T, /turf/space))
+				boutput(user, SPAN_ALERT("What exactly are you gunna secure [src] to?"))
```

Comment:
"gunna" instead of "gonna"

Reply:
I actually copy-pasted that message from the fountain and thought I shouldn't change it because it was already written like that.